lib/server.js

```diff
@@ -21,35 +21,37 @@ module.exports = {
   serverRoutes: {},
   serverPolicies: {},
   serverHandlers: {},
+  webConfig: {},
 
   createServer (app) {
     const config = app.config
     const server = express()
-    this.port = config.web.port
-    this.host = config.web.host
-    this.ssl = config.web.ssl
-    this.redirectToHttps = config.web.redirectToHttps || false
+    this.webConfig = _.cloneDeep(app.config.web)
```

Comment:
Yeah I start to move stuff on `configure` and then stop because of this because I would have to create the express server on the `configure` method witch is not good.